config/boards/orangepi5-plus.wip

```diff
@@ -0,0 +1,34 @@
+# Rockchip RK3588 octa core 4/8/16GB RAM SoC NVMe USB3 USB-C 2.5GbE eMMC
+BOARD_NAME="Orange Pi 5 plus"
+BOARDFAMILY="rockchip-rk3588"
+BOOTCONFIG="orangepi_5_plus_defconfig" # vendor name, not standard, see hook below, set BOOT_SOC below to compensate
+BOOT_SOC="rk3588"
+KERNEL_TARGET="legacy"
+FULL_DESKTOP="yes"
+BOOT_LOGO="desktop"
+BOOT_FDT_FILE="rockchip/rk3588-orangepi-5-plus.dtb"
+BOOT_SCENARIO="spl-blobs"
+BOOT_SUPPORT_SPI="yes"
+BOOT_SPI_RKSPI_LOADER="yes"
+IMAGE_PARTITION_TABLE="gpt"
+SKIP_BOOTSPLASH="yes" # Skip boot splash patch, conflicts with CONFIG_VT=yes
+BOOTFS_TYPE="ext4"
```

Comment:
Is booting from nvme with spinor working?

Reply:
tested it's working.